docs/guide.adoc

```diff
@@ -302,7 +303,7 @@ end
 
 `new_player` creates a `Player` with a built in update-function that
 only mutates the created `Player` blob. The `Player` blob is baked
-into the update-function and cannot be changed after creation.
+into the update-function as the constant `self`.
```

Comment:
```suggestion
into the update-function as the psuedo-variable `self`.
```
I feel like claiming `self` to be constant is a bit confusing, since you can change the fields (which you can do with constants, but I'd say it's not the main point)

Reply:
pseudo-variable is on the other hand a bit vague. I'd remove it all together.

```
The `Player` blob is baked
into the update-function as `self`.
```

The example makes it clear how it can be used.